android/upload_sourcemap.sh

```diff
@@ -39,6 +39,25 @@ else
     echo "Instabug: Generating sourcemap files..."
     IS_HERMES=$(grep "enableHermes:" ./android/app/build.gradle -m 1)
     if [[ $IS_HERMES == *"true"* ]]; then
+        #Find HERMES OS bin directory
+        case "$OSTYPE" in
+            darwin*)  HERMES_OS_BIN='osx-bin' ;; 
+            linux*)   HERMES_OS_BIN='linux64-bin' ;;
+            msys*)    HERMES_OS_BIN='win64-bin' ;;
+            *)        echo "unknown: $OSTYPE" ;;
+        esac
+
+        #Find HERMES command file name
+        RN_VERSION=$(npm view react-native version)
```

Comment:
I believe this command gets the most recent version regardless of the one installed. I was thinking of a more direct approach. What do you think?
```
INSTALLED_RN_VERSION=`node -p "require('<pkg_path_in_node_modules>/package.json').version"`
```

Reply:
True, that command is more accurate. Thanks, I'll use it.